### PR TITLE
Adds comments to exported functions

### PR DESCRIPTION
**Pull request by gabzim**

### CODE REVIEW

File: example-app/src/app.ts [hunk 1]
```diff
@@ -52,6 +52,11 @@ async function main() {
   app.use(stagingArea());
   app.use(sourceTx());
 
+  app.multiple(
```
Comment (JoeHowarth) [inline example-app/src/app.ts]: Shouldn't we either have this line or 60? What happens if there are overlapping routes?

File: relayer/application.ts [hunk 2]
```diff
@@ -196,26 +235,44 @@ export class RelayerApp<ContextT extends Context> {
...
+   * @param chainId
+   */
   chain(chainId: ChainId): ChainRouter<ContextT> {
     if (!this.chainRouters[chainId]) {
       this.chainRouters[chainId] = new ChainRouter(chainId);
     }
     return this.chainRouters[chainId]!;
   }
 
+  /**
```
Comment (JoeHowarth) [inline relayer/application.ts:254]: Meta: should middlewares be able to extend the app? This seems like a good convenience function that should become available iff the token bridge middleware is being used. Extending the app seems like an interesting extension point.

File: relayer/application.ts [hunk 3]
```diff
@@ -236,25 +293,54 @@ export class RelayerApp<ContextT extends Context> {
     return Array.from(spyFilters.values());
   }
 
+  /**
+   * Pass in the URL where you have an instance of the spy listening. Usually localhost:7073
+   *
+   * You can run the spy locally by doing:
```
Comment (JoeHowarth) [inline relayer/application.ts]: Specify this command is for testnet

**Review by JoeHowarth — CHANGES_REQUESTED** [review]

File: example-app/src/app.ts (hunk 1 above)
Comment (gabzim) [reply example-app/src/app.ts]: if there are overlapping routes, all of them get added as a middleware, maybe we could think about this.

**Pull request merged**